features/support/publishing_api_helpers.rb

```diff
@@ -15,6 +15,15 @@ def assert_content_item_is_published_to_publishing_api(base_path)
         "publishing_app" => "policy-publisher",
       },
     )
+
+    assert_publishing_api_put_item(
```

Comment:
Rather than make this part of the existing "And a policy called "BLAH" is published to publishing API" step, I think it would be better to have an explicit separate step for this, e.g. "And an email alert signup page is created for the policy". That makes the fact more visible in the policy features.